provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,81 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
+
+## Setup within ManageIQ
+
+### Set up the workflow repository
```

Comment:
Lines 9 and 11, can be probably combined as a single line with title:

## Set up the workflow repository within ManageIQ